packages/esm-patient-registration-app/src/patient-registration/input/custom-input/autosuggest/autosuggest.test.tsx

```diff
@@ -82,17 +82,17 @@ describe('autosuggest', () => {
     expect(handleSuggestionSelected).toHaveBeenNthCalledWith(1, 'person', 'randomuuid1');
   });
 
-  it.skip('sets search box value to selected suggestion', async () => {
-    setup();
-    let searchbox = screen.getByRole('searchbox');
-    fireEvent.change(searchbox, { target: { value: 'john' } });
-    const listitems = await waitFor(() => screen.getAllByRole('listitem'));
-    fireEvent.click(listitems[0]);
-    searchbox = screen.getByRole('searchbox');
-    expect(searchbox.textContent).toBe('John Doe');
-  });
+  // it('sets search box value to selected suggestion', async () => {
+  //   setup();
+  //   let searchbox = screen.getByRole('searchbox');
+  //   fireEvent.change(searchbox, { target: { value: 'john' } });
+  //   const listitems = await waitFor(() => screen.getAllByRole('listitem'));
+  //   fireEvent.click(listitems[0]);
+  //   searchbox = screen.getByRole('searchbox');
+  //   expect(searchbox.textContent).toBe('John Doe');
+  // });
```

Comment:
can we fix this test? @ayush-AI

Reply:
@anjula-sack I don't think that we need this test because if auto-suggest component doesn't behave this way if you will click on any of the suggested item it will send you to that patient summary page. Below is the example video

https://user-images.githubusercontent.com/54752747/235086810-0aeed65b-7342-4cf5-8ffc-43f699e7b327.mov

Reply:
yeah, @ayush-AI we can remove that test.